src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/SampleSequencingObjectFileModel.java

```diff
@@ -0,0 +1,42 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto;
+
+import ca.corefacility.bioinformatics.irida.model.sequenceFile.SequencingObject;
```

Comment:
I thought we were going to move the ajax and dto's within the directory of their respective controller?

Reply:
Yup that's the plan :)